modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/view_dashboard.jsp

```diff
@@ -0,0 +1,19 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+ViewDashboardDisplayContext viewDashboardDisplayContext = (ViewDashboardDisplayContext)request.getAttribute(ViewDashboardDisplayContext.class.getName());
+%>
+
+<div class="cms-section">
+	<react:component
+		module="{ViewDashboard} from site-cms-site-initializer"
+		props="<%= viewDashboardDisplayContext.getReactData() %>"
```

Comment:
Just a suggestion: Can you name this display context method `getProps()` so that it matches the attribute name?

Reply:
For sure!

Reply:
Hello @adolfopa, there is actually a pattern in all JSP files that uses the nomenclature `getReactData`. I think we should keep the same.

Reply:
Sounds good.